tests/conftest.py

```diff
@@ -127,10 +127,13 @@ def _register_handler(self, path: Path) -> str:
 
         return self._httpserver.url_for(f"/{path.name}")
 
-    def add_wheel(self, path: Path):
+    def add_wheel(self, path: Path, replace: bool = True):
         name = parse_wheel_filename(path.name)[0]
         url = self._register_handler(path)
 
+        if name in self._wheels and not replace:
+            return
```

Comment:
Maybe we should raise here? I think silently doing nothing is likely to be confusing.

Reply:
I used this method to register all the wheels included in the Pyodide distribution as test wheels, as without this, they would conflict with pytest wheel, etc that I put in for testing.

I think it will be fine as long as we know what we are doing. I added some comments about it.